teuthology/task/install.py

```diff
@@ -644,6 +701,17 @@ def _remove_rpm(ctx, config, remote, rpm):
     :param remote: the teuthology.orchestra.remote.Remote object
     :param rpm: list of packages names to remove
     """
+    
+    if config.get('rhbuild'):
+        log.info("Uninstalling packages using ceph-deploy")
+        r = remote.run(args=['date'],check_status=False)
+        host = r.hostname
+        remote.run(args = ['sudo', 'ceph-deploy', 'purge', host])
+        remote.run(args = ['sudo', 'ceph-deploy', 'uninstall', host])
+        log.info("Uninstalling ceph-deploy")
+        remote.run(args = ['sudo', 'yum', 'remove', 'ceph-deploy', '-y'],check_status=False)
+        return True
```

Comment:
Wait, so if 'rhbuild' is in ``config`` you do something completely different?

I'm starting to wonder if this should be a separate task entirely, instead of heavily modifying the install task...